ci/format.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+ROOT_DIR=$SCRIPT_DIR/..
+
+cd $ROOT_DIR || exit $?
+
+EXCLUDES=$(tr '\n' ',' < .gitignore)
+EXCLUDES="$EXCLUDES,.*/migrations"
+IGNORES="E402" # module level import not at top of file (breaks how settings include local settings)
+
+if [ "$1" == "--fix" ]; then
+    autopep8 --ignore=$IGNORES --in-place --recursive --exclude $EXCLUDES . || exit $?
+fi
```

Comment:
Perhaps it's easier to write a single configuration file than to construct command-line arguments.
See https://pypi.org/project/autopep8/#configuration.

Reply:
Good idea, fixed that now. Also put a bit more thought into this, extended the max line length and made some manual (but stable) tweaks. Pretty happy with this PR now! Thanks a lot for pushing to get this done.

Reply:
I can btw wait with merging this until I've had time to review and merge your other PR, then you don't have to rebase on formatting changes.

Reply:
No need to block the PR, merge as you wish. I'll rebase if necessary.

Reply:
Cool, thanks a lot! I'll merge now then.